front/src/applications/editor/tools/routeEdition/components/RouteMetadata.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { RouteEntity } from 'types';
+
+export const RouteMetadata: FC<{
+  entity: RouteEntity;
+  disabled: boolean;
+  disableTransit: boolean;
+  onChange: (e: boolean) => void;
+}> = ({ entity, onChange, disableTransit, disabled }) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="form-group" style={{ opacity: disabled ? 0.5 : 1 }}>
+      <div className="row">
+        <span className="col-sm-6 col-form-label">
+          {t('Editor.tools.routes-edition.start_direction')}
+        </span>
+        <div className="col-sm-6">
+          {!disabled && (
+            <span>
+              {t(
+                `Editor.tools.routes-edition.directions.${entity.properties.entry_point_direction}`
+              )}
+            </span>
+          )}
+        </div>
```

Comment:
Maybe we can avoid rendering the div if `disabled`

```suggestion
     {!disabled && (
        <div className="col-sm-6">          
            <span>
              {t(
                `Editor.tools.routes-edition.directions.${entity.properties.entry_point_direction}`
              )}
            </span>
        </div>
        )}
```